lagom13-scala-service-locator-dns/src/main/scala/com/lightbend/lagom/scaladsl/dns/DnsServiceLocatorComponents.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright © 2016 Lightbend, Inc. All rights reserved.
+ * No information contained herein may be reproduced or transmitted in any form
+ * or by any means without the express written permission of Typesafe, Inc.
+ */
+
+package com.lightbend.lagom.scaladsl.dns
+
+import akka.actor.ActorRef
+import com.lightbend.dns.locator.{ ServiceLocator => ServiceLocatorService }
+import com.lightbend.lagom.scaladsl.client.CircuitBreakerComponents
+
+/**
+ * Provides the DNS service locator.
+ */
+trait DnsServiceLocatorComponents extends CircuitBreakerComponents {
```

Comment:
OK, but again, the conductr-lib ones that @jroper wrote also inherit from `CircuitBreakerComponents`...